src/screens/ProfileScreen/index.js

```diff
@@ -280,9 +273,10 @@ const ProfileScreen = ({route}) => {
     try {
       setIsFetchingList(true);
       setIsHitApiFirstTime(true);
+      const cacheFeed = StorageUtils.myFeeds.get();
       const result = await getSelfFeedsInProfile(offset, limit);
       const {data: dataMyFeed} = result;
-      const mapNewData = mappingColorFeed(dataMyFeed);
+      const {mapNewData} = mappingColorFeed({dataFeed: dataMyFeed, dataCache: cacheFeed});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for fetching feeds has been updated to include caching. Make sure the cache is properly invalidated when necessary to avoid displaying stale data.
```diff
+ const cacheFeed = StorageUtils.myFeeds.get();
+ const {mapNewData} = mappingColorFeed({dataFeed: dataMyFeed, dataCache: cacheFeed});
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->